bin/init.bash

```diff
@@ -12,29 +12,29 @@ here="$(dirname "$(readlink -f "$0")")"
 source "${here}/common.bash"
 
 # Load cloud provider, environment name, and flavor from config if available.
-if [ -f "${config[default_config_file_sc]}" ]; then
-    cloud_provider=$(yq read "${config[default_config_file_sc]}" 'global.ck8sCloudProvider')
-    environment_name=$(yq read "${config[default_config_file_sc]}" 'global.ck8sEnvironmentName')
-    flavor=$(yq read "${config[default_config_file_sc]}" 'global.ck8sFlavor')
+if [ -f "${config[default_common]}" ]; then
+    cloud_provider=$(yq read "${config[default_common]}" 'global.ck8sCloudProvider')
+    environment_name=$(yq read "${config[default_common]}" 'global.ck8sEnvironmentName')
+    flavor=$(yq read "${config[default_common]}" 'global.ck8sFlavor')
```

Comment:
I've added to the readme that these are set by `init` and are needed to generate the correct config during upgrades, and how they can be changed if needed by removing the file.
Since these are not intended to be changed or overriden by the user directly I don't think it should be an issue just checking the default common config.